src/printer.js

```diff
@@ -656,6 +656,7 @@ function genericPrintNoParens(path, options, print) {
                   join(concat([ ",", line ]), path.map(print, "elements"))
                 ])
               ),
+              n.elements[n.elements.length - 1] === null ? "," : "",
```

Comment:
Don't you need to check if `n.elements.length > 0` ? What's the AST like for `[,]`? (I can check myself tomorrow)